src/libs/actions/Session/index.ts

```diff
@@ -882,6 +878,23 @@ const canAccessRouteByAnonymousUser = (route: string) => {
     return false;
 };
 
+/**
+ * navigate to exitTo path after signin
+ *
+ * @param {Routes | HybridAppRoute} exitTo
+ */
+
+const handleExitToNavigation = (exitTo: Routes | HybridAppRoute) => {
```

Comment:
This sounds generic. I think we can keep the same name `signInWithValidateCodeAndNavigate` since it already has the navigation part in it and update the docs to say it looks for exitTo query param and navigates accordingly.

Reply:
After fully looking into the code, I don't think this is doing what I expected, so maybe this is fine. But we can update the method name to give more context `handleExitToNavigationAfterSignIn`? What you think about it?